src/main/java/decodes/tsdb/comprungui/CompRunGuiFrame.java

```diff
@@ -760,6 +780,69 @@ private void runButtonPressed()
 			return;
 		}
 
+		TsGroup compGroup = isSelectionValid(compVector);
+
+		if (compGroup == null)
+		{
+			Logger.instance().debug3("None of the selected computations uses a group.");
```

Comment:
```suggestion
			Logger.instance().debug3("None of the selected computations uses a group.");
			return;
```
compGroup gets passed as argument below, not sure if null is supported.

Reply:
Completely changed the way this works.